timelock-impl/src/main/java/com/palantir/atlasdb/timelock/AsyncTimelockServiceImpl.java

```diff
@@ -106,6 +112,12 @@ public ListenableFuture<LockResponseV2> lock(IdentifiedLockRequest request) {
                 request.getLockDescriptors(),
                 TimeLimit.of(request.getAcquireTimeoutMs()),
                 request.getMetadata());
+        metadataMetrics
```

Comment:
This will construct a _new_ histogram object for every request; we should create the histogram `metadataMetrics::numChangeMetadata` in the constructor

Reply:
Also fixed this for the counters